src/components/Exchange/constants.ts

```diff
@@ -0,0 +1,11 @@
+export enum ErrorCodes {
```

Comment:
We recently discussed with G, enums should be called in the singular for consistency: ErrorCodes -> ErrorCode

You can also capitalize field names - ErrorCodes.Buffer

Reply:
Thanks. Updated